src/components/include/security_manager/security_manager.h

```diff
@@ -120,18 +125,34 @@ class SecurityManager : public protocol_handler::ProtocolObserver {
    * identifier
    * @return new \c  SSLContext or \c NULL on any error
    */
-  virtual SSLContext* CreateSSLContext(const uint32_t& connection_key) = 0;
+  virtual SSLContext* CreateSSLContext(const uint32_t& connection_key,
```

Comment:
@AByzhynar please update description

Reply:
@LuxoftAKutsan Changed in eb15dd852a0ea418bfa444b74c8f15d792e254ad